model/Software/Properties/location.md

```diff
@@ -0,0 +1,16 @@
+# location
+
+## Summary
+
+TODO
+
+## Description
+
+A location is TODO
+
+## Metadata
+
+- name: location
+- Nature: DataProperty
+- Range: SnippetLocation
```

Comment:
Will every custom datatype will become Vocab at some time?

Reply:
Every _enumeration_ will be a vocabulary.